src/welle-gui/i18n/es_ES.ts

```diff
@@ -0,0 +1,1766 @@
+        <translation>Albanés</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="230"/>
+        <source>Breton</source>
+        <translation>Bretón</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="231"/>
+        <source>Catalan</source>
+        <translation>Catalán</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="232"/>
+        <source>Croatian</source>
+        <translation>Croata</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="233"/>
+        <source>Welsh</source>
+        <translation>Galés</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="234"/>
+        <source>Czech</source>
+        <translation>Checo</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="235"/>
+        <source>Danish</source>
+        <translation>Danés</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="236"/>
+        <source>German</source>
+        <translation>Alemán</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="237"/>
+        <source>English</source>
+        <translation>Inglés</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="238"/>
+        <source>Spanish</source>
+        <translation>Español</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="239"/>
+        <source>Esperanto</source>
+        <translation>Esperanto</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="240"/>
+        <source>Estonian</source>
+        <translation>Estonés</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="241"/>
+        <source>Basque</source>
+        <translation>Vasco</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="242"/>
+        <source>Faroese</source>
+        <translation>Feroés</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="243"/>
+        <source>French</source>
+        <translation>Francés</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="244"/>
+        <source>Frisian</source>
+        <translation>Frisón</translation>
+    </message>
+    <message>
+        <location filename="../../backend/dab-constants.cpp" line="245"/>
+        <source>Irish</source>
+        <translation>Irlanés</translation>
```

Comment:
Typo? But I do not speak Spanish

Reply:
Hi Andreas,

Yes, it is a typo. It should be "Irlandés". File updated.

Regards,
Nándor